google-cloud-storage/samples/storage_print_pubsub_bucket_notification.rb

```diff
@@ -0,0 +1,40 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START storage_print_pubsub_bucket_notification]
+def print_pubsub_bucket_notification bucket_name:, notification_id:
+  # The ID of your GCS bucket
+  # bucket_name = "your-unique-bucket-name"
+
+  # The ID of your notification configured for the bucket
+  # notification_id = "your-notification-id"
+
+  require "google/cloud/storage"
```

Comment:
I think we should move the require outside the method (but still inside the region tags). It's much more common for requires to be executed once at the top of a file, outside the class/method definitions, than for them to be executed inside a method like this.

Reply:
True that, have updated it for this sample. We might have to update other samples as well to reflect the same.